components/Header/SecondaryNav/index.stories.tsx

```diff
@@ -1,54 +1,33 @@
+import MockBlock from '../MockBlock'
 import SecondaryNav from '.'
-import type { Meta, Story } from '@storybook/react'
+import type { Meta } from '@storybook/react'
 import type { ReactElement } from 'react'
 
-const style = {
-  backgroundColor: 'rgb(245,245,245)',
-  border: '2px dashed rgb(220,220,220)',
-  color: 'rgb(170,170,170)',
-  fontSize: 'large',
-  height: '18rem',
-  lineHeight: '18rem',
-  textIndent: '2ch',
-}
-
-const Template: Story<Record<string, never>> = (args) => <SecondaryNav {...args} />
-
 const meta: Meta = {
   component: SecondaryNav,
   decorators: [
     (Story: () => ReactElement): JSX.Element => {
       return (
         <>
-          <header style={style}>Header</header>
+          <MockBlock>Header</MockBlock>
           <Story />
-          <main>
-            <section style={style} id="section-one">
-              Section 1
-            </section>
-            <section style={style} id="section-two">
-              Section 2
-            </section>
-            <section style={style} id="section-three">
-              Section 3
-            </section>
-            <section style={style} id="section-four">
-              Section 4
-            </section>
-          </main>
+          <MockBlock id="section-one">Section 1</MockBlock>
+          <MockBlock id="section-two">Section 2</MockBlock>
+          <MockBlock id="section-three">Section 3</MockBlock>
+          <MockBlock id="section-four">Section 4</MockBlock>
         </>
       )
     },
   ],
   parameters: {
     a11y: {
-      element: '#root header ~ *:not(main)',
+      element: '#root *:not(.mockBlock, .mockBlock *)',
```

Comment:
I think the `:not()` pseudo-class is not working here. Should probably look like the following if we are wanting to exclude those `MockBlock` components :grin:
```
element: '#root :not([class*="mockBlock"])'
```